cadence/transactions/change_forwarder_recipient.cdc

```diff
@@ -0,0 +1,35 @@
+import NonFungibleToken from "../contracts/NonFungibleToken.cdc"
+import ExampleNFT from "../contracts/ExampleNFT.cdc"
+import NFTForwarding from "../contracts/utility/NFTForwarding.cdc"
+
+/// This transaction is what an account would run
+/// to change the NFTForwarder recipient
+
+transaction(newRecipientAddress: Address) {
+
+    /// Reference to the NFTFowarder Resource
+    let forwarderRef: &NFTForwarding.NFTForwarder
+    /// Collection we will designate as forwarding recipient
+    let newRecipientCollection: Capability<&{NonFungibleToken.CollectionPublic}>
+
+    prepare(signer: AuthAccount) {
+        /// Borrow reference to NFTForwarder resource
```

Comment:
Comments within functions should only be two slashes so we don't confuse the doc generator tool